tldw_Server_API/app/core/AuthNZ/federation/oidc_service.py

```
@@ -0,0 +1,345 @@
+        resolved = {
+            "issuer": _coerce_nonempty_string(provider.get("issuer")) or _coerce_nonempty_string(discovery.get("issuer")),
+            "authorization_url": _coerce_nonempty_string(provider.get("authorization_url"))
+            or _coerce_nonempty_string(discovery.get("authorization_endpoint")),
+            "token_url": _coerce_nonempty_string(provider.get("token_url"))
+            or _coerce_nonempty_string(discovery.get("token_endpoint")),
+            "jwks_url": _coerce_nonempty_string(provider.get("jwks_url"))
+            or _coerce_nonempty_string(discovery.get("jwks_uri")),
+            "client_id": _coerce_nonempty_string(provider.get("client_id")),
+            "client_secret": await _resolve_client_secret_ref(provider.get("client_secret_ref")),
+        }
+        return resolved
```

Ruff (0.15.6)

[warning] Unnecessary assignment to `resolved` before `return` statement

Remove unnecessary assignment

(RET504)